modules/ossm-routing-bookinfo-example.adoc

```diff
@@ -4,168 +4,14 @@
 // * service_mesh/v2x/ossm-traffic-manage.adoc
 
 [id="ossm-routing-bookinfo_{context}"]
-= Routing example using the bookinfo application
+= Routing tutorial
 
-The {ProductShortName} Bookinfo sample application consists of four separate microservices, each with multiple versions. Three different versions, one of the microservices called `reviews`, have been deployed and are running concurrently.
+The {ProductShortName} Bookinfo sample application consists of four separate microservices, each with multiple versions. After installing the Bookinfo sample application, three different versions have been deployed and are running concurrently.
```

Comment:
```suggestion
When you install the {ProductShortName} Bookinfo sample application, it deploys four microservices. Each microservice consists of three pods, one for each version of the microservice.
```
Is this what it is saying?

Reply:
I don't think we need to be that specific. For the tutorial, it's important you know that there are three different versions of the `reviews` microservice. In the tutorial, you switch between the versions to show you how to route traffic. I think the pods aren't totally relevant here.

Reply:
But I understand what you're saying. I'm going to change it to "three different versions of the 'reviews' microservice were deployed and are running concurrently."